.github/workflows/prerelease.yml

```diff
@@ -88,14 +88,18 @@ jobs:
         LLVM_EXTERNAL_LIT: "/usr/local/bin/lit"
 
     steps:
+      - name: Clean Docker to prevent space issues
+        run: |
+          docker system prune -af --volumes || true
+
       - name: Clone the Patchestry repository
         uses: actions/checkout@v4
         with:
           submodules: true
           fetch-depth: 1
 
       - name: Configure build
-        run: cmake --preset ci
+        run: cmake --preset ci -DUSE_VENDORED_CLANG=1 -DLLVM_Z3_INSTALL_DIR=/usr/local
```

Comment:
I believe this should allow us to pick up the vendored llvm20, but it might not be needed if one already exists in the runtime container

Reply:
Should we use vended clang here? Only concern I have is related to time. It takes time and consume CI minutes to build clang. The workflow will kick-off with pr merge.

Reply:
Yeah I hear that - I'm going to try building with `act` without this and see what happens!!

Reply:
Looks like it works just fine without the vendored Clang, so I've pushed that commit